ps_searchbar.js

```diff
@@ -3,20 +3,38 @@ $(document).ready(function () {
     var $searchWidget = $('#search_widget');
     var $searchBox    = $searchWidget.find('input[type=text]');
     var searchURL     = $searchWidget.attr('data-search-controller-url');
+    var $clearButton  = $searchWidget.find('i.clear');
 
     $.widget('prestashop.psBlockSearchAutocomplete', $.ui.autocomplete, {
         _renderItem: function (ul, product) {
+            var $img = $('<div class="autocomplete-thumbnail">');
+            if (product.cover) {
+              $img.append('<img src="'+product.cover.bySize.small_default.url+'">')
+            } else {
+              $img.append('<img src="'+prestashop.urls.no_picture_image.bySize.small_default.url+'">')
+            }
```

Comment:
Looks like this could be made simpler:

```js
const coverImage = (product.cover) ? product.cover : prestashop.urls.no_picture_image;

return $('<li>')
  .append($('<a>'))
     .append($('<img>').attr('src', coverImage.bySize.small_default.url))
     .append(/* ... */)
```